src/components/Tag/Tag.tsx

```diff
@@ -24,6 +24,8 @@ export interface Props extends React.HTMLAttributes<HTMLDivElement> {
     contentStyle?: React.CSSProperties;
```

Comment:
I see a few props (cannot comment on lines above) that aren't supported now in the new design: isSelected, maxWidth, size. maybe we don't need both onDelete and onClick?

I checked the usage in JD, we use onClick, not onDelete so that when the whole tag is clicked it still deletes the label. We also use `size` but we can get away without it I think.

Could you check current usage with the Search team on their end?
Maybe we can delete these props, or maybe we should keep them (we have a design for selectable tags, but it was outscoped in this ticket). Maybe check with Dima too, and when you have all the info, we can make a decision together.

Also, it would be good to adjust the stories so that they reflect the current use cases. It would be good to have one story for `read-only` tag, so without an onClick/onDelete callback, one with the callBack and one for coloured version.

Reply:
I looked at JD and in LabelTag we use both onClick and onDelete. We also use MaxWidth, and size indeed. I don't know if we need to avoid breaking changes ;)  
Tag is not used in Search at all, apparently! So that makes it a bit easier...

Reply:
ok, let's keep the props as they are then